iam_policy/main.tf

```diff
@@ -1,11 +1,16 @@
+module "tags" {
```

Comment:
Update the policy's README.md with proper formatting (tables, examples, etc.)

Reply:
Still waiting on this

Reply:
The readme has been updated. Can you not see it? Or is there a specific problem with the formatting?

Reply:
Sorry, I'm an idiot. Can you update the formatting of all tf files?

Reply:
Updated 👍